GitUI/Lemmings.cs

```diff
@@ -15,15 +15,12 @@ public static Bitmap GetPictureBoxImage(DateTime currentDate)
                 if (currentDate.Month == 12 && currentDate.Day > 18 && currentDate.Day < 27) //X-Mass
                 {
                     return Resources.Cow_xmass;
-                } else
-                    if (currentDate.Month == 6 && currentDate.Day == 21) //summer
-                    {
-                        return Resources.Cow_sunglass;
-                    }
-                    else
-                    {
-                        return Resources.Cow;
-                    }
+                }
+                if (currentDate.Month == 6 && currentDate.Day == 21) //summer
+                {
+                    return Resources.Cow_sunglass;
+                }
+                return Resources.Cow;
```

Comment:
Removed braces here.

Reply:
I applied manually the Resharper rule to remove the useless `else`